internal/controller/postgrescluster/instance.go

```diff
@@ -1200,25 +1200,33 @@ func (r *Reconciler) reconcileInstance(
 			spec, instanceCertificates, instanceConfigMap, &instance.Spec.Template)
 	}
 
-	if err == nil &&
-		(feature.Enabled(ctx, feature.OpenTelemetryLogs) && !feature.Enabled(ctx, feature.OpenTelemetryMetrics)) {
+	if err == nil {
```

Comment:
@dsessler7 We had talked about reworking this logic a bit along these lines, here's my pass at it.

Note: this work is done with the understanding that we're not rotating postgres or patroni logs through logrotate right now.